### ISSUE TITLE
Policy retention rules dryrun logs empty

### ISSUE THREAD

**Issue by ricardojdsilva87** [goharbor/harbor#20771]
Hello, we are experiencing the same situation.
We are running Harbor v2.11.0 but on v2.10.x we experienced the same situation that the logs are empty even on dryruns on the projects.
When trying to do it to check which images would be deleted, the log is always empty, even after a run, the page is still blank.
We already tried to change the logs to DEBUG, but nothing changes.
Is there any setting that needs to be updated?
Thanks

Related issue:
https://github.com/goharbor/harbor/issues/18505

**Comment by zyyw** [comment]
Are you using db logging? It so, there is a similar issue documented here:
* https://github.com/goharbor/harbor/pull/20549

And it's now fixed by a PR.

**Comment by ricardojdsilva87** [comment]
That's correct @zyyw we have enabled the database option.
We'll wait then for a new release to test the fix :)
Thanks!

**Comment by ricardojdsilva87** [comment]
Hello again @zyyw,
I've tried changing the type from `database` to `file` and it remained the same.
After that I've tried to update the loglevel to INFO and it seems to work. We have tried this in versions previous to 2.11.0 and it seemed that it was not working, but we might have made some mistakes testing it.
We had enabled only the warning level for logs since we have them being exported to another platform and the information in them was not useffull for us on daily basis.
For now I've updated the log level to INFO only on the jobservice configmap and left the remaining still on WARNING.
We'll check how can we leave the configuration without removing it with new deployments.
We are using the official Harbor Helm Chart:
https://github.com/goharbor/harbor-helm/tree/main

Thanks
